manifest.yml

```diff
@@ -10,7 +10,8 @@ applications:
   env:
     JBP_CONFIG_OPEN_JDK_JRE: '{jre: { version: 11.+ }}'
     APP_VERSION: ((app-version))
+    ELIGIBILITY_URI: http://htbhf-eligibility-service((app-suffix)).apps.internal:8080
```

Comment:
This needs to be space-suffix, not app-suffix. app-suffix includes '-green'

Reply:
Sorry, copy-paste error, done.